File: app/src/main/java/net/radityalabs/contactapp/presentation/ui/activity/TestActivity.java
package net.radityalabs.contactapp.presentation.ui.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.annotation.VisibleForTesting;
import android.support.v7.app.AppCompatActivity;
import android.widget.FrameLayout;

import net.radityalabs.contactapp.R;

/**
 * Created by radityagumay on 4/19/17.
 */

@VisibleForTesting
public class TestActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable final Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        FrameLayout frameLayout = new FrameLayout(this);
        frameLayout.setId(R.id.container);
        setContentView(frameLayout);
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/ui/activity/TestActivity.kt
package net.radityalabs.contactapp.presentation.ui.activity

import android.os.Bundle
import android.support.annotation.VisibleForTesting
import android.support.v7.app.AppCompatActivity
import android.widget.FrameLayout

import net.radityalabs.contactapp.R

/**
 * Created by radityagumay on 4/19/17.
 */

@VisibleForTesting
class TestActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val frameLayout = FrameLayout(this)
        frameLayout.id = R.id.container
        setContentView(frameLayout)
    }
}